site/en/docs/devtools/open/index.md

```diff
@@ -55,5 +55,7 @@ Mac:
 /Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome --auto-open-devtools-for-tabs
 ```
 
+Will only work if an instance of Chrome is not already running. From then on, every new tab will automatically open DevTools until user fully quits Chrome <kbd>Command</kbd>+<kbd>Q</kbd>.
```

Comment:
```suggestion
This will only work if an instance of Chrome is not already running. From then
on, every new tab will automatically open DevTools until the user fully quits
Chrome.
```

Reply:
@jpmedley CLA was signed on Apr 04, 2021 00:53 PDT.